Document initialOutputMedium and transcriptOptional (#39)
fixie-ai/ultradox: docs/src/content/docs/api/calls.mdx

        <td class="font-mono">initator</td>
        <td>string</td>
        <td>Who was responsible for starting this call. Typically set to INITIATOR_AGENT for outgoing calls and left as the default (INITIATOR_USER) otherwise.</td>
      </tr>
       <tr>
        <td class="font-mono">initialOutputMedium</td>
        <td>string</td>

>         <td>The initial medium (MESSAGE_MEDIUM_VOICE or MESSAGE_MEDIUM_TEXT) to use for the call. Defaults to MESSAGE_MEDIUM_VOICE. Can be changed by using [`setOutputMedium`](../../sdk#setoutputmedium) from the Ultravox client SDK.</td>

Reviewer: ```suggestion
        <td>The initial medium (MESSAGE_MEDIUM_VOICE or MESSAGE_MEDIUM_TEXT) to use for the call. Defaults to MESSAGE_MEDIUM_VOICE. Once the call has started, the output medium can be changed by the client using [`setOutputMedium`](../../sdk#setoutputmedium).</td>
```

        <td>string</td>
        <td>Message that the agent will say if the call reaches the `maxDuration`.</td>
      </tr>
      <tr>
        <td class="font-mono">transcriptOptional</td>
        <td>boolean</td>

>         <td>Defaults to `true`. Setting to `false` (not recommended) enables live, user transcripts which may not match what the model hears. Instead, we recommend using [List Call Message](#list-call-messages) to get accurate user transcripts after the call completes. </td>

Reviewer 2: ```suggestion
        <td>Defaults to `true`. Setting to `false` (not recommended) enables live, user transcripts at the expense of latency. Furthermore, these transcripts may not match what the model actually hears.</td>
```
